cli/command/image/build_session.go

```diff
@@ -17,9 +17,9 @@ import (
 	"github.com/docker/cli/cli/command/image/build"
 	cliconfig "github.com/docker/cli/cli/config"
 	"github.com/docker/docker/api/types/versions"
-	"github.com/docker/docker/client/session"
-	"github.com/docker/docker/client/session/filesync"
 	"github.com/docker/docker/pkg/progress"
+	"github.com/moby/buildkit/session"
+	"github.com/moby/buildkit/session/filesync"
```

Comment:
Let's keep this for a separate PR. This code is still in `docker/docker/client` in master, so we shouldn't need to do this right now.

Reply:
It is not there anymore on moby/moby master :(

Reply:
oops, I thought I checked before commenting, but it must have been an old branch. ok to ignore this one